xbmc/guilib/GUITexture.cpp

```diff
@@ -71,15 +71,14 @@ void CGUITexture::DrawQuad(const CRect& coords,
 
 CGUITexture::CGUITexture(
     float posX, float posY, float width, float height, const CTextureInfo& texture)
-  : m_height(height), m_info(texture)
+  : m_diffuseColor(0xffffffff), m_height(height), m_info(texture)
```

Comment:
there is a reason why you are moving all PR's members here?
its a compire warning?
if not, please restore to original position